anta/input_models/routing/isis.py

```diff
@@ -122,3 +122,63 @@ def __init__(self, **data: Any) -> None:  # noqa: ANN401
             stacklevel=2,
         )
         super().__init__(**data)
+
+
+class SRTunnelEntry(BaseModel):
```

Comment:
We are already using `Segment` + `TunnelPath` so I think `Tunnel` makes sense.
```suggestion
class Tunnel(BaseModel):
```

Reply:
You also need to add a ClassVar for the deprecated `Vias` model for backward compatibility.

Reply:
Updated input model, added deprecation warning for Vias model as well. Thanks!!